### PR TITLE
Pkg.autobahn cpp

### PR DESCRIPTION
**Pull request by tatraian**
Disabled tests that are not passed. All have reason:)...

### CODE REVIEW

File: cmake/configs/default.cmake [hunk 1]
```diff
@@ -90,6 +90,7 @@ hunter_config(ZLIB VERSION 1.2.8-p3)
 hunter_config(ZMQPP VERSION 4.1.2)
 hunter_config(ZeroMQ VERSION 4.1.4-p2)
 hunter_config(caffe VERSION rc3-p2)
+hunter_config(autobahn-cpp VERSION 0.2.0)
```
Comment (ruslo) [inline cmake/configs/default.cmake]: This one doesn't belong this pull request.

**Review by ruslo — CHANGES_REQUESTED** [review]

File: cmake/configs/default.cmake (hunk 1 above)
Comment (tatraian) [reply cmake/configs/default.cmake]: Sorry I don't understand. Probably my whole PR is bad. I just merged my `autobahn-cpp` into your `pkg.autobahn-cpp`. This didn't contains any autobahn specific code, so they were added.
So what I change?

Comment (ruslo) [reply cmake/configs/default.cmake]: You should not merge your regular branches with ingenue/hunter branches. Workflow is `develop` -> `pkg.template` -> `pkg.*` -> `upload.*`. You should not "jump" from `develop` directly to `pkg.*`. The rule is simple if you're making pull request to `pkg.*` branch you have to check that it contains only `.travis.yml`/`appveyor.yml` modifications. Just like top-level README of this repository states:
- https://github.com/ingenue/hunter/

**Comment by tatraian** [comment]
Only CI config modification arrived.

**Review by ruslo — APPROVED** [review]

**Pull request merged**